### PR TITLE
Read/write timeseries section of inp

### PR DESCRIPTION
**Pull request by jackieff**
#57 Addresses TIMESERIES section. Model object can now read timeseries section (direct data and file-pointers). `replace_inp_section` also works. Added a file-pointing timeseries into `model_full_features_network.inp` for testing.

### CODE REVIEW

File: swmmio/core.py [hunk 1]
```diff
@@ -924,7 +926,7 @@ def polygons(self):
     @polygons.setter
     def polygons(self, df):
         """Set inp.polygons DataFrame."""
-        self._curves_df = df
+        self._polygons_df = df
```
Comment (aerispaha) [inline swmmio/core.py:929]: Oh wow is this a bonus bug fix?

File: swmmio/tests/test_model_elements.py [hunk 2]
```diff
@@ -162,3 +162,9 @@ def test_example_1():
     peak_runoff = model.rpt.subcatchment_runoff_summary['PeakRunoff']
     assert peak_runoff.values == pytest.approx([4.66, 4.52, 2.45, 2.45, 6.56, 1.5, 0.79, 1.33], rel=0.001)
     assert peak_runoff.values == pytest.approx(subs['PeakRunoff'].values, rel=0.001)
+
+def test_get_set_timeseries(test_model_01):
+
+    ts = test_model_01.inp.timeseries
```
Comment (aerispaha) [inline swmmio/tests/test_model_elements.py:168]: Can we assert something here?

**Review by aerispaha — CHANGES_REQUESTED** [review]
This rules, thanks @jackieff! 

My only request is that we add an actual assertion in the `test_get_set_timeseries` unit test

File: swmmio/core.py (hunk 1 above)
Comment (jackieff) [reply swmmio/core.py:929]: Think so!

File: swmmio/tests/test_model_elements.py (hunk 2 above)
Comment (jackieff) [reply swmmio/tests/test_model_elements.py]: Done!

**Review by aerispaha — APPROVED** [review]
Cool!

**Pull request merged**